var/spack/repos/builtin/packages/cbtf-argonavis/package.py

```diff
@@ -67,6 +63,40 @@ class CbtfArgonavis(Package):
 
     parallel = False
 
+    build_directory = 'build_cbtf_argonavis'
+
+    def build_type(self):
+        return 'None'
```

Comment:
We don't like the compile options provided by CMake, so we are defining our own for our tool.